app/static/js/picslider.js

```diff
@@ -67,17 +67,41 @@ function update_pic_slider(data){
 }
 
 function init_pic_slider(data){
+  if (filterJSParams['selected_time'] == "ALL"){
+    var para = document.createElement("p")
+    var bold = document.createElement("strong")
+    var br = document.createElement("br")
+    var bold_2 = document.createElement("strong")
+    var text_1 = document.createTextNode("View a different face from a different time by selecting the ")
+    var text_2 = document.createTextNode("Timespan ")
+    var text_3 = document.createTextNode("control or by selecting the ")
+    var text_4 = document.createTextNode("Dimension ")
+    var text_5 = document.createTextNode("control which filters on age, gender or color group.")
+    var text_6 = document.createTextNode("Have fun!")
```

Comment:
Have fun??

Reply:
What is this though?

Reply:
javascript to make parapgraphs and strong fields